auto_editor/StructuredProjectSource.py

```diff
@@ -39,12 +39,20 @@ def get_all_dpct_file_paths(self) -> List[str]:
         :return: list with paths to all files inside directory at self.dpct_version_root
         '''
         # Todo: use same logic here as in PreAnalyser. Currently nested files aren't discovered.
-
+        dpct_extensions = ('*.dp.cpp', '*.dp.hpp')
```

Comment:
nice! you can even move this to the constants.py file which already exists. Using a separate file for constants is a common practice, so that if something changes, later on it can all be done in one place.